scenarios4/15_The_Library.cfg

```diff
@@ -606,7 +614,9 @@
             result=defeat
         [/endlevel]
     [/event]
+#ifndef HARD
     {DROPS 5 8 (sword,bow,staff,sword,bow) no 2,3}
```

Comment:
No DROPS defined on hard? I am not sure how well will it work.

Reply:
You're already picking up like 6 books in less than 10 turns.